host-interaction/process/inject/allocate-rwx-memory.yml

```diff
@@ -21,6 +21,8 @@ rule:
         - api: kernel32.VirtualProtect
```

Comment:
should we remove the `att&ck` mapping as this general functionality is not necessarily process injection?

Reply:
Fixed in aa94a33